concourse/pipeline.py

```diff
@@ -51,6 +51,7 @@ def update_version(new_version, source, target):
 
         @job.task()
         def install_and_test():
+            shell(["python3", "-m", "pip", "install", "-r", "requirements.txt"], cwd="pipeline-dsl")
```

Comment:
Why did you decide to use `python3 -m pip install` instead of `pip3 install`?
Don't we have `pip3` installed directly?
For me, it doesn't matter I'm just asking out of curiosity.

Reply:
Personally, I ran often in issues that `pip`/`pip3` used a different python version than my actual `python`/`python3` and the package was not found. With the `-m pip` it worked and so far I haven't found any downside.